src/legacy/shared/GoogleMapsWithMarkers/GoogleMapsWithMarkers.tsx

```diff
@@ -11,12 +11,16 @@ import { LatLngBounds, ListingShort } from 'networking/listings';
 
 import GoogleMapsWithMarkersContainer from './GoogleMapsWithMarkers.container';
 
+const hotelMarker = require('assets/images/iconmonstr-location-12-32.png');
```

Comment:
i think we can upload these to s3 and use it there

Reply:
either way, the icons are still cached via cloudfront.